### PR TITLE
Update FAQ about cleaning downloads and build trees

### PR DESCRIPTION
**Pull request by mloskot**
Answers #2202 and #2538

### CODE REVIEW

File: docs/about/faq.md [hunk 1]
```diff
@@ -72,6 +72,10 @@ A lower level mechanism to achieve the same as the `vcpkg integrate project` NuG
 <Import Project="<vcpkg_root>\scripts\buildsystems\msbuild\vcpkg.targets" />
 ```
 
+## How can I remove downloads and build trees?
```
Comment (KindDragon) [inline docs/about/faq.md]: Maybe call it: "How can I remove temporal files"?

Comment (mloskot) [reply docs/about/faq.md]: Done

**Comment by ras0219-msft** [comment]
Thanks for the PR!

**Comment by mloskot** [comment]
Happy to help

**Pull request merged**